testing/gmock/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=gmock
+pkgver=1.8.0
+pkgrel=0
+pkgdesc="Google Mock - A library for writing and using C++ mock classes"
+url="https://github.com/google/googletest"
+arch="all"
+license="BSD"
+depends="libgcc bash python2 gtest"
+depends_dev="python2 cmake"
+makedepends="$depends_dev autoconf automake libtool coreutils"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="gtest-$pkgver.tar.gz::https://github.com/google/googletest/archive/release-$pkgver.tar.gz
+  gmock-install.patch
+  gmock.pc.in
+"
+builddir="$srcdir"/googletest-release-${pkgver}
+
+prepare() {
+  cd "$builddir"
+  find -name '*.py' -exec sed -i 's|env python|env python2|g' {} \;
+  default_prepare || return
+  cd googlemock
+  cp "${srcdir}/gmock.pc.in" .
+  autoreconf -fvi || return
+}
+
+build() {
+  cd "$builddir"
+  cd googlemock
+  ./configure --prefix=/usr --with-gtest --enable-external-gtest || return
+  make || return
+}
+
+package() {
+  cd "$builddir"
+  cd googlemock
+  make --trace -w -j1 DESTDIR="${pkgdir}" install || return
```

Comment:
This installs even files that should be moved into -doc subpackage:

/usr/share/gmock/generator/LICENSE
/usr/share/gmock/generator/README
/usr/share/gmock/generator/README.cppclean